src/index.html

```diff
@@ -4,9 +4,7 @@
 	<meta charset="utf-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
-	<% if (process.env.NODE_ENV === "production" ) { %>
-		<meta http-equiv="Content-Security-Policy" content="script-src 'self' 'unsafe-inline' 'unsafe-eval' blob; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com/;  img-src 'self';worker-src 'self' blob: ">
-	<% } %>
+	
```

Comment:
If a `CSP` is required, we have the option of putting it into the Tauri config, but it's SO nice to maybe get this removed.